docs/reference/intro.asciidoc

```diff
@@ -155,81 +184,146 @@ You can use a combination of dynamic and explicit mapping on the same index.
 This is useful when you have a mix of known and unknown fields in your data.
```

Comment:
fair enough here I guess it might be "unexpected" for the naive user, so it's more of a statement about a novice user's epistemology — "which you might find unexpected" — but agree this can be tightened. I don't love _imprecise_ either. We also don't need to be exhaustive here.

I'd rather be concrete like "can lead to suboptimal mappings because certain field types might not be inferred correctly".

> re: explicit on like 150 ... it's not just about control, it's also about making your data more functional and opening up additional use cases.

ok but we link to the main mappings section for details, this is just an overview at end of the day, we don't need 100% exhaustive summaries of capabilities 

in my little head "control" is a decent encapsulation of the advantage of explicit mappings in this context but happy to reword

Reply:
went with 

- "This is great for getting started quickly, but can yield suboptimal results for complex data due to automatic field type inference."
- "Recommended for production use cases, because you have full control over how your data is indexed to suit your specific use case."